biome.json

```diff
@@ -68,22 +68,14 @@
       "node_modules",
       "docker",
       "bundle.js",
-      "website/blog",
-      "website/build",
```

Comment:
I'd actually keep this one, if you don't mind. There's no point running biome on the generated output, I think.  Even if that's an edge case.

Reply:
Good catch, for some reason I thought it would build to the `.docusaurus` folder